flow-model-generator/modules/flow-model-index-generator/src/main/java/io/ballerina/indexgenerator/SearchListGenerator.java

```diff
@@ -47,7 +47,7 @@ class SearchListGenerator {
     private static final Map<String, List<String>> SKIPPED_PACKAGE_LIST = Map.of(
             "ballerina", List.of("xmldata", "regex", "persist", "grpc"),
             "ballerinax", List.of("persist.googlesheets", "persist.inmemory", "persist.redis", "persist.sql",
-                    "health.hl7v2.utils.v2tofhirr4", "netsuite", "ai.agent", "np"));
+                    "health.hl7v2.utils.v2tofhirr4", "netsuite", "ai.agent"));
```

Comment:
This change is already done in a previous PR

Reply:
Could you please point that PR here

Reply:
https://github.com/ballerina-platform/ballerina-language-server/pull/70/files#diff-532308ceddcd3467b66495e3435503a2e2529906096aeff5ce15a0901ee35f94